### PR TITLE
[DONT MERGE] [IE TESTS INFRA] Actual & expected tensor comparation fix

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[IE TESTS INFRA] Actual & expected tensor comparation doesn't work.*

### Tickets:
 - *CVS-133173*

### CODE REVIEW

File: src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp [hunk 1]
```diff
@@ -426,52 +426,57 @@ void compare(const ov::Tensor& expected,
     auto max_type_actual = std::numeric_limits<ActualT>::max();
     auto min_type_expected = std::numeric_limits<ExpectedT>::min();
     auto min_type_actual = std::numeric_limits<ActualT>::min();
-    Error abs_error(abs_threshold), rel_error(rel_threshold);
-    for (size_t i = 0; i < shape_size_cnt; ++i) {
+    double abs_value = 0.0;
```
Comment (iefode) [inline src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp:429]: `Error` structure contains logic based on eps. Could you please revert `double` changes? Error structure accumulate the general error. In other case, please explain details why you decided to change

File: src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp [hunk 2]
```diff
@@ -426,52 +426,57 @@ void compare(const ov::Tensor& expected,
...
+    double rel_value = 0.0;
+    size_t i = 0;
+    for (; i < shape_size_cnt; ++i) {
```
Comment (iefode) [inline src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp:432]:
```suggestion
for (size_t i = 0; i < shape_size_cnt; ++i) {
```

File: src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp [hunk 3]
```diff
@@ -426,52 +426,57 @@ void compare(const ov::Tensor& expected,
...
-        rel_error.update(rel, i);
+        if ((rel_value > rel_threshold) || (abs_value > abs_threshold)) {
+            break;
+        }
     }
-    abs_error.mean /= shape_size_cnt;
-    rel_error.mean /= shape_size_cnt;
 
-    if (!(less_or_equal(abs_error.max, abs_threshold) || less_or_equal(rel_error.mean, rel_threshold))) {
+    if ((rel_value > rel_threshold) || (abs_value > abs_threshold)) {
```
Comment (iefode) [inline src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp:474]:
```suggestion
    if (!(less_or_equal(abs_error.max, abs_threshold) || less_or_equal(rel_error.mean, rel_threshold))) {
```

File: src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp [hunk 4]
```diff
@@ -73,8 +73,8 @@ class SubgraphBaseTest : public ov::test::TestsCommon {
 
     std::function<void(const std::exception& exp)> callback_exception = nullptr;
 
-    constexpr static const double disable_threshold = std::numeric_limits<double>::max();
-    double abs_threshold = disable_threshold, rel_threshold = disable_threshold;
+    constexpr static const double default_threshold = 0.01;
```
Comment (sbalandi) [inline src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp:76]: default_threshold = 0

Comment (sbalandi) [inline src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp:76]: default_threshold = 0; or should be changed default thresholds in compare() to 0.01

**Pull request closed without merging**